apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { CheckboxField, TextInput, Pill, Button } from '@contentful/forma-36-react-components';
+
+import { styles } from './styles';
+
+async function callAPI(url) {
+  const data = await fetch(`http://localhot:3000/tags/${url}`).then(res => res.json());
```

Comment:
don't mix `async/await` with promises

Reply:
In the final version we can just change it to `/tags/...` since both frontend and backend are served from the same domain

Reply:
On the mixing part - mostly avoiding another line of `await res.json()`. I can chnage it tho